content/logs/log_collection/docker.md

```diff
@@ -112,7 +112,7 @@ logs:
 ```
 
 When filtering on the container image, both exact container image name or short names are supported.
-Suppose you have one container running `library/httpd:latest` the following filtering match this image name:
+If you have one container running `library/httpd:latest`, the following filtering match this image name:
```

Comment:
If you have one container running `library/httpd:latest`, the following filtering match**es** this image name: